tests/run/shadow_compatibility.pyx

```diff
@@ -0,0 +1,24 @@
+# mode: run
+
+import cython
+
+
+def check_int(s):
+    """
+    >>> check(5)
+    True
+    >>> check("Hello world!")
+    False
+    """
+    return isinstance(s, cython.int)
+
+
+# FIXME: broken, see #5521
+# def check_str(s):
+#     """
+#     >>> check(5)
+#     False
+#     >>> check("Hello world!")
+#     True
+#     """
+#     return isinstance(s, cython.str)
```

Comment:
I don't think there is a `cython.str`, is there? Only `unicode` should be special because it does not exist in Py3.